src/ui/Facet/FacetSettings.ts

```diff
@@ -232,9 +236,13 @@ export class FacetSettings extends FacetSort {
   }
 
   private enabledSortsAllowDirection() {
-    return _.some(this.enabledSorts, (facetSortDescription: IFacetSortDescription) => {
+    const allEnabledSortsWithPossibleDirectionToggle = _.filter(this.enabledSorts, (facetSortDescription: IFacetSortDescription) => {
       return facetSortDescription.directionToggle;
     });
+    const allowToggle = _.filter(allEnabledSortsWithPossibleDirectionToggle, (possibleDirectionToggle) => {
+      return _.findWhere(this.enabledSorts, { name: possibleDirectionToggle.relatedSort }) != null;
```

Comment:
Hum yes, technically you are right.

But since it's not a ===, `null == undefined`

Still, it's cleaner to change it, yeah.